README.md

```diff
@@ -1,11 +1,70 @@
-# SYCL-Docs
-SYCL Open Source Specification
+![SYCL Logo](latex/logos/SYCL_RGB_June16-inkscape-1500.png)
+
+![Build Status](https://api.travis-ci.com/KhronosGroup/SYCL-Docs.svg?branch=master)
+[![SPEC master](https://img.shields.io/badge/SPEC-master-red.svg?logo=adobe-acrobat-reader)](https://khronosgroup.github.io/SYCL-Docs/sycl/sycl.pdf)
+[![SPEC 1.2.1-5](https://img.shields.io/badge/SPEC-1.2.1--5-orange.svg?logo=adobe-acrobat-reader)](https://www.khronos.org/registry/SYCL/specs/sycl-1.2.1.pdf)
+[![Join the Slack group](https://img.shields.io/badge/chat-on%20slack-blue.svg?logo=slack)](https://www.khronos.org/news/permalink/khronos-developer-slack-5bfc62eb261764.20435008)
```

Comment:
I've changed the link to the invite link. This was set to the blog post first because it contains both the invite link and the actual link to the Slack.